README.md

```diff
@@ -3,7 +3,7 @@
 A website that showcases projects! **If you want to use it and it's not exactly
 fitting your purpose, please do make upstream changes. We expect this project
 to be of use to others and we're happy to make it more generic so it's easier
-to adapt to other organizations.**
+to adapt to other organizations.** To view live projects visit projects.coala.io
```

Comment:
👍  It'll be good to link it, check this out https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

It'll look like
```
To view live projects visit [some-text](projects.coala.io)
```